pygmo/docstrings.cpp

```diff
@@ -1602,7 +1602,7 @@ The adaptation of the mutation variant is added to :class:`~pygmo.core.sade`
 
 Args:
     gen (``int``): number of generations
-    allowed_variants (``NumPy array or list of floats``): allowed mutation variants, each one being a number in [1, 18]
+    allowed_variants (``NumPy array or list of int``): allowed mutation variants, each one being a number in [1, 18]
```

Comment:
I guess any iterable looking at the  ```pygmo::to_vu``` code.
Looking at the docs for fitness, there we say (array-like object) is that ok to refer to any python iterable object?